client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionDialog.tsx

```diff
@@ -122,8 +124,12 @@ const InteractionDialog = (props: Props) => {
             [InteractionEventDialogFields.END_TIME]: endTimeToSave,
             [InteractionEventDialogFields.UNKNOWN_TIME]: Boolean(data[InteractionEventDialogFields.UNKNOWN_TIME]),
             [InteractionEventDialogFields.ID]: methods.watch(InteractionEventDialogFields.ID),
-            [InteractionEventDialogFields.PLACE_NAME]: Boolean(data[InteractionEventDialogFields.PLACE_NAME]) ?
-                data[InteractionEventDialogFields.PLACE_NAME] : generatePlacenameByPlaceSubType(placeSubtypeName),
+            [InteractionEventDialogFields.PLACE_NAME]: 
+                isNamedLocation 
+                    ? Boolean(data[InteractionEventDialogFields.PLACE_NAME])
+                        ? data[InteractionEventDialogFields.PLACE_NAME]
+                        : generatePlacenameByPlaceSubType(placeSubtypeName)
+                    : generatePlacenameByPlaceSubType(placeSubtypeName),
```

Comment:
I think you can check if isNamedLocation and Boolean(..) then data[..] else generate, instead of writing 2 ifs

Reply:
oh yea, fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1428/commits/753603cf8155765aeb63646cf62453871caf7262)